### DISCUSSION TITLE
Help revive JobFunnel

### DISCUSSION THREAD

**Discussion by anoduck** [PaulMcInnis/JobFunnel#163] [General]
#### Current Status:

**A solution to the captcha issue might have been discovered.** As, it no longer prevents generation of the indeed url creation.

#### What needs to be done:

- [ ] Update bs4 selectors for indeed.
- [ ] Update bs4 selectors for other sites.
- [ ] Test to ensure captcha is no longer interfering with scraping.
- [ ] Do the happy dance.

#### The Spiel

I have worked on updating the selectors for bs4 a little, but I would feel more comfortable working alongside someone who is more experienced with python.

**Those interested may notate below.**

**Comment by PaulMcInnis** [comment]
Working on this now, will use playwright, will keep the discussion in the other thread. 

Thanks for your interest, feel free to test when I have a new version ready if you're still around 🫡
